core-bundle/contao/themes/flexible/styles/layout/list-view.pcss

```
@@ -285,7 +285,8 @@
 }
 
 .operations a,
-.operations button {
+.operations button,
+.operations span.disabled {
```

```suggestion
.operations span {
```

Could technically allow it for spans as well (in case there'll be more in the future)

---

I thought about that, but wasn't sure if that would possibly affect any existing element styling.

---

There isn't really custom styling within the operations, could however leave it as it is right now :) Feel free to resolve